docs/source/getting-started/content/ref_minimum-system-requirements.adoc

```diff
@@ -0,0 +1,40 @@
+[id="minimum-system-requirements_{context}"]
+= Minimum system requirements
+
+{prod} requires the following minimum hardware and operating system requirements.
+
+[id="minimum-system-requirements-hardware_{context}"]
+== Hardware requirements
+
+{prod} requires the following system resources:
+
+* 4 virtual CPUs (vCPUs)
+* 8 GB of memory
+* 35 GB of storage space
+
+[NOTE]
+====
+These requirements must be met in order to run OpenShift in the {prod} virtual machine.
+Other applications running on top of OpenShift may require additional resources to run.
```

Comment:
This is actually for sure needed. 'Depending on the workload more resources need to be assigned'